seshdash/utils/reporting.py

```diff
@@ -213,3 +216,27 @@ def get_table_report_dict(report_table_name, operations):
                 report_table_attributes.append(report_attribute_dict)
 
     return report_table_attributes
+
+
+"""
+Kapacitor report utils
+"""
+def add_report_kap_task():
+    """  
+    This function adds a report task to kapacitor
+    """
+    data = {
```

Comment:
It was for testing ,Now the function accepts a report instance and  pulls the data from the report instance.